src/DataViews/include/DataViews/LiveFunctionsDataView.h

```diff
@@ -61,6 +60,13 @@ class LiveFunctionsDataView : public DataView {
 
   std::string GetToolTip(int /*row*/, int column) override;
 
+  void AddToIndices(ScopeId scope_id) {
```

Comment:
As it is a public method now, maybe call is `AddScope`?

Reply:
Done.